### PR TITLE
Update Harvard-Cannon environment for latest software library updates *in /n/jacob_lab/Lab/...)

### PR DESCRIPTION
**Pull request by yantosca**
### Name and Institution (Required)

Name: Bob Yantosca
Institution: Harvard + GCST

### Describe the update

We have updated the files in `envs/Harvard-Cannon` to be consistent with the latest software updates in the `/n/jacob_lab/Lab` folder.  Specifically,

- Environment files `gcclassic*env` are taken from the [cannon-env](https://github.com/Harvard-ACMG/cannon-env) version2 2.0.0
- The `imi_env.yml` is now based on the recent GCPy 1.4.0 environment, plus extra Python packages needed for the IMI.  The environment now builds in one shot with MambaForge.
- The `config.harvard-cannon.yml` file now points to the updated paths (i.e. `/n/jacob_lab/Lab/` instead of `/n/jacob_lab/Lab/seasasfs02/Lab`, etc.)

This will be a zero-diff update as it only affects configuration files specific to Harvard.

### CODE REVIEW

File: envs/Harvard-Cannon/config.harvard-cannon.yml [hunk 1]
```diff
@@ -119,24 +119,24 @@ DataPath: "/n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/gcdata/ExtData"
...
 DataPathTROPOMI: "/n/holylfs05/LABS/jacob_lab/lshen/CH4/TROPOMI/data"
 
-## Environment files
+## Environment files (edit paths as necessary)
 ## See envs/README to create the Conda environment specified below
-NCOEnv: "/n/home05/msulprizio/GC/CH4/integrated_methane_inversion/envs/Harvard-Cannon/gcc.ifort17_cannon.env"
-GEOSChemEnv: "/n/home05/msulprizio/GC/CH4/integrated_methane_inversion/envs/Harvard-Cannon/gcc.gfortran10.2_cannon.env"
-CondaEnv: "ch4_inv"
+NCOEnv: "/n/jacob_lab/Lab/cannon-env/envs/gnu10/gcclassic.rocky+gnu10.env"
+GEOSChemEnv: "/n/jacob_lab/Lab/cannon-env/envs/gnu10/gcclassic.rocky+gnu10.minimal.env"
```
Comment (msulprizio) [inline envs/Harvard-Cannon/config.harvard-cannon.yml:125]: These two paths should point to the env files in the IMI repo -- `envs/Harvard-Cannon/gcclassic.rocky+gnu12.minimal.env`. Note the switch to using gnu12 by default too.

**Review by msulprizio — CHANGES_REQUESTED** [review]

File: envs/Harvard-Cannon/config.harvard-cannon.yml (hunk 1 above)
Comment (yantosca) [reply envs/Harvard-Cannon/config.harvard-cannon.yml]: Thanks @msulprizio.  Now fixed in 511852a.

File: envs/Harvard-Cannon/imi_env.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+# $ mamba env create -n imi_em --file=/path/to/gcpy/imi_env.yml
+#
+# Some package versions are not the most recent, but these
+# have been proven to work together. (Bob Yantosca, 31 Aug 2023)
+#
+# NOTE: This environment uses packages that have been confirmed
+# to work with GCPy.  Additional are listed near end of this file.
+# ======================================================================
+name: imi_env
+channels:
```
[the remark this answers is not in the document]
Comment (yantosca) [reply envs/Harvard-Cannon/imi_env.yml]: Thanks @laestrada.  I've restored the prior `ch4_inv.yml` file in commit c8b9ce33.

**Review by laestrada — APPROVED** [review]

**Review by msulprizio — APPROVED** [review]

**Comment by yantosca** [comment]
@laestrada: I have rebased the `feature/rocky-linux-envs` branch up to main (imi-1.2.1) in commit s 2d4595 and de55d8e.   I've also restored the original `imi_env.yml` file

**Comment by yantosca** [comment]
Closing this PR as this brings in several older updates.  Will do a fresh PR off of the main branch.  Apologies for the bother.,

**Pull request closed without merging**